my-app/src/AddMeal.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import "./AddMeal.css"
+
+function AddMeal() {
+    const [clicked, setclicked] = useState(false)
+    const [addedMeals, setAddedMeals] = useState({
+        title: "",
+        description: "",
+        location: "",
+        when: "",
+        price: "",
+        max_reservations: ""
+    })
+    const handelMealName = (e) => {
+        setAddedMeals({...addedMeals, title: e.target.value})
+    }
+    const handelMealDescription = (e) => {
+        setAddedMeals({...addedMeals, description: e.target.value})
+    }
+    const handelLocation = (e) => {
+        setAddedMeals({...addedMeals, location: e.target.value})
+    }
+    const handelDate = (e) => {
+        setAddedMeals({...addedMeals, when: e.target.value})
+    }
+    const handelMealPrice = (e) => {
+        setAddedMeals({...addedMeals, price: e.target.value})
+    }
+    const handelMaxReservation = (e) => {
+        setAddedMeals({...addedMeals, max_reservations: e.target.value})
+
+    }
+    const handelAddMeal = () => {
+        setclicked(true)
+        fetch("http://localhost:5000/api/meals",
+        {
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            method: "POST",
+            body: JSON.stringify(addedMeals)
+        })
+        .then(function(res){ console.log(res) })
+        .catch(function(res){ console.log(res) })
+        setAddedMeals({
+            title: "",
+            description: "",
+            location: "",
+            when: "",
+            price: "",
+            max_reservations: ""})
```

Comment:
Nice with resetting the state :) you could move it inside the `.then(function(res){ "HERE" })` then it would only reset the state if it succeeds :)  but up to you